egs/librimix/ConvTasNet/eval.py

```diff
@@ -41,11 +53,101 @@
 parser.add_argument(
     "--n_save_ex", type=int, default=10, help="Number of audio examples to save, -1 means all"
 )
+parser.add_argument(
+    "--compute-wer", type=int, default=0, help="Compute WER using ESPNet's pretrained model"
+)
+
+COMPUTE_METRICS = ["si_sdr", "sdr", "sir", "sar", "stoi"]
+ASR_MODEL_PATH = (
```

Comment:
Will need to be a variable probably.

Reply:
it should be specified by the user according to the recipe

Reply:
Yes, and we'll have to make sure it was trained on LibriSpeech.

Reply:
I assumed that ASR models couldn't be used to perform cross datasets evaluation but is this true ?

Reply:
There is the problem of vocab at least. If the vocab of the second dataset is included in the first one, then that's probably possible. Samu can probably answer better than me.